cmd/rqlite/main.go

```diff
@@ -127,7 +129,20 @@ func sendRequest(ctx *cli.Context, urlStr string, line string, argv *argT, ret i
 
 	nRedirect := 0
 	for {
-		resp, err := client.Post(url, "application/json", strings.NewReader(data))
+		req, err := http.NewRequest("POST", url, strings.NewReader(data))
+		if err != nil {
+			return err
+		}
+
+		if argv.Credentials != "" {
+			user, pass, err := net.SplitHostPort(argv.Credentials)
+			if err != nil {
+				return fmt.Errorf("invalid Basic Auth credentials format")
```

Comment:
Why is the error being discarded and not returned to the user.. should be `fmt.Errorf("inv..... %v", err)`

Reply:
Because nothing useful would come from that error. Since I'm using the `SplitHostPort`, if the user would do `-u bob` (no password) this would raise the following message: `missing port in address`.

Reply:
Then why not use `errors.New` since this would produce an error if ran through `go vet`

Reply:
or so I think

Reply:
`go vet` is ok with that 👍

Reply:
👍